src/deroff.rs

```diff
@@ -838,29 +838,18 @@ impl Deroffer {
         unimplemented!()
     }
 
-    fn macro_tr(&self) -> bool {
-        unimplemented!()
-    }
-
-    /*
-    handled by Kevin
-    fn macro_so_nx(&mut self) -> bool {
-        /*  # We always ignore include directives
-            # deroff.c for some reason allowed this to fall through to the 'tr' case
-            # I think that was just a bug so I won't replicate it */
-        true
-    }
-
-    handled by Anders
+    
     fn macro_tr(&mut self) -> bool {
-        let s = self.s.clone();
-        let s = s.as_str();
-        self.s = self.skip_char(s, Some(2)).to_owned();
-        self.s = self.skip_leading_whitespace(s).to_owned();
-        while !self.s.is_empty() && Self::str_at(s, 0) != "\n" {
-            let c = Self::str_at(s, 0);
-            let mut ns = Self::str_at(s, 1);
-            self.s = self.skip_char(self.s.as_str(), Some(2)).to_owned();
+        let s = &self.s.clone();
+        // self.skip_char(2);
+        self.s = self.skip_char(&self.s, 2).to_owned();
```

Comment:
I believe this is outdated.

Reply:
Hmm. We got rid of that clone before merging. Not sure why Github isn't labeling this code snippet as outdated.